Source/ASDisplayNode.mm

```diff
@@ -1304,6 +1310,7 @@ + (void)scheduleNodeForRecursiveDisplay:(ASDisplayNode *)node
     }];
   });
 
+  as_log_verbose(ASRenderLog(), "%s %@", sel_getName(_cmd), node);
```

Comment:
I think ASDisplayLog might be a better name for this, as we've standardized on Display Range, recursivelyEnsureDisplay, setNeedsDisplay, etc. There are some stragglers like renderQueue here though, and I realize ASDisplay looks a bit like ASDisplayNode, so you can decide.